src/pydocstyle/config.py

```diff
@@ -208,7 +208,10 @@ def _get_config(self, node):
         if path in self._cache:
             return self._cache[path]
 
-        config_file = self._get_config_file_in_folder(path)
+        if self._options.config_file:
```

Comment:
Yeah, the specifics of `--config` are more complicated that I expected and #237 works well for me. Someone else can pick this up if they feel so inclined. ✌️